apps/fabric-website/src/utilities/location.ts

```diff
@@ -3,10 +3,15 @@
  * has the Universal Header and Footer (UHF).
  */
 export const hasUHF: boolean =
-  window.location.hostname === 'developer.microsoft.com' || window.location.hostname === 'developer.microsoft-tst.com';
+  typeof window === undefined
```

Comment:
I don't think this works.

`typeof window` would return a string, so this would always resolve as false.

Reply:
Yes, it needs to be `typeof window === 'undefined'` (with the quotes around undefined)